person.rb

```diff
@@ -0,0 +1,22 @@
+class Person 
+  attr_reader :id , :name , :age
```

Comment:
- [ ] As an OOP convention, it's impractical to assign different accessor attributes to the same variable in a particular class. `name` and `age` variable should have only one accessor attribute, and thus we need to modify and read `name` and `age` variables and therefore they should be marked with  `attr_accessor`. You can read further on the same via this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)

Reply:
@rloterh Thank you for this tip!